shared/styles/index.d.ts

```diff
@@ -161,6 +161,5 @@ export type {
 export {default as classNames} from 'classnames'
 export {default as styled} from '@emotion/styled'
 export declare const CanFixOverdrawContext: React.Context<boolean>
-export declare const dontFixOverdraw: {canFixOverdraw: false}
```

Comment:
unrelatedly the overdraw context is just a single value so lets skip with the issues w/ holding a whole object